app/containers/Applications/ApplicationWizard.tsx

```diff
@@ -42,6 +43,11 @@ export const ApplicationWizardComponent: React.FunctionComponent<Props> = ({
     latestSubmittedRevision,
     applicationRevisionByStringVersionNumber: applicationRevision
   } = application;
+  const reviewComments = query.application.reviewCommentsByApplicationId?.edges.map(
+    ({node: description}) => description
+  );
+  const revisionInProgress =
+    latestSubmittedRevision?.versionNumber < latestDraftRevision.versionNumber;
```

Comment:
would `latestDraftRevision.versionNumber > 1`  work too?

Side not. I got very confused because "review" and "revision" are very similar words.

Reply:
What I was probably thinking is that the above would cover an edge case where they were viewing this page (from a pasted link) with `version` equal to the latest submitted version. 

But of course this page actually redirects them to view-application, so that wouldn't happen. I suppose both comparisons are about equal in that case.